functions/index.js

```diff
@@ -35,3 +17,20 @@ exports.notifySubmission = functions.firestore
     notifySlack(msg);
     return true;
   });
+
+exports.testEmail = functions.https.onRequest(async (req, res) => {
```

Comment:
this looks really dangerous to me: an unauthenticated https endpoint that sends email (using a real smtp server by the way, as noted in another comment) is posed to be hacked. I gather from the name of the email is here just for testing purposes?

Reply:
yes, that's why it's called "testEmail" just to show how it works... I'm gonna comment this out as you can see it working now